### DISCUSSION TITLE
Main menu volume option

### DISCUSSION THREAD

**Discussion by iamzomb** [Quaver/Quaver#4794] [Feature Request]
### Describe the solution you'd like

I'd just like an additional sound slider to adjust the main menu volume. Me and a couple friends I play with all agree that we'd like to have the main menu music be quieter than when in an actual song. Thanks

### Describe alternatives you've considered

Well all I can really do at the moment is change the overall music volume until I want to start a song then change it, which is annoying. (all the love, everything else is awesome but just saying how I feel about this.)

**Comment by AiAe** [comment]
This has been implemented in Canary (1.8)
